integration/testrunner/private_key.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Just put the plain key in this file

Reply:
If we just read all the bytes from this file and write it to the header, we need it to be in this format so the code works.